share/spack/setup-env.sh

```diff
@@ -284,6 +284,7 @@ else
     _sp_share_dir="$(cd "$(dirname $_sp_source_file)" > /dev/null && pwd)"
     _sp_prefix="$(cd "$(dirname $(dirname $_sp_share_dir))" > /dev/null && pwd)"
 fi
+
```

Comment:
Vestigial space? I assume this is based on some edit that was later reverted.

Reply:
Yes, good catch.